docs/source/local_model.md

```diff
@@ -54,6 +54,23 @@ The most frequently used parameters are `model`, `max_token`, and `temperature`.
 Please check what you can set for the model parameter
 at [LlamaIndex LLM](https://docs.llamaindex.ai/en/latest/api_reference/llms.html).
 
+📌 Of course, you can use `custom LLM` hosted privately!
```

Comment:
It is confusing docs. Just add api_base parameter at above 'openailike' yaml file example.